<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quiz</class>
 <widget class="QWidget" name="Quiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/images/zoom.svg</normaloff>:/img/images/zoom.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 243, 250);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,3">
     <item>
      <widget class="QPushButton" name="pushButton_title">
       <property name="styleSheet">
        <string notr="true">padding: 20px;
border-radius: 20px;
  min-height: 20px;
  min-width: 20px;
background-image: url(:/background/images/bg/intro bg.png);
font: 24pt &quot;Audiowide&quot;;
font: bold;
color:white;</string>
       </property>
       <property name="text">
        <string>Dictionary - Group 2</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,2,0,0">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_goBack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 6px;
border-color: rgb(77, 101, 246);
border-width : 1px;
border-style:inset;
border-radius: 8px;
  min-height: 1px;
  min-width: 1px;
background-color: rgb(102, 124, 246);
font: 11pt &quot;Comfortaa&quot;;
font: bold;
color: white;</string>
         </property>
         <property name="text">
          <string>Back </string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/images/back.svg</normaloff>:/img/images/back.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 5px;
border-color: rgb(77, 101, 246);
border-width : 1px;
border-style:inset;
border-radius: 8px;
  min-height: 1px;
  min-width: 1px;
color: white;
selection-color: rgb(39, 182, 240);
selection-background-color: rgb(77, 92, 245);
font: 11pt &quot;Comfortaa&quot;;
font: bold;
background-color: rgb(102, 124, 246);</string>
         </property>
         <item>
          <property name="text">
           <string>1 Definition - 4 Keywords</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 Keyword - 4 Definitions</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_newQuiz">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 6px;
border-color: rgb(77, 101, 246);
border-width : 1px;
border-style:inset;
border-radius: 8px;
  min-height: 1px;
  min-width: 1px;
background-color: rgb(102, 124, 246);
font: 11pt &quot;Comfortaa&quot;;
font: bold;
color: white;</string>
         </property>
         <property name="text">
          <string>New Quiz</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/images/new.svg</normaloff>:/img/images/new.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_ques">
       <property name="styleSheet">
        <string notr="true">padding: 10px;
border-width : 2px;
border-style:solid;
border-radius: 20px;
  min-height: 20px;
  min-width: 20px;
background-color: rgb(233, 240, 250);
border-color: rgb(99, 112, 243);
font: 13pt &quot;Comfortaa&quot;;
color: rgb(56, 64, 139);</string>
       </property>
       <property name="text">
        <string>Ques</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="1,1">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_c1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 20px;
border-width : 2px;
border-style:solid;
border-radius: 20px;
  min-height: 20px;
  min-width: 20px;
background-color: rgb(233, 240, 250);
border-color: rgb(99, 112, 243);
font: 13pt &quot;Comfortaa&quot;;
color: rgb(56, 64, 139);</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_c3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 20px;
border-width : 2px;
border-style:solid;
border-radius: 20px;
  min-height: 20px;
  min-width: 20px;
background-color: rgb(233, 240, 250);
border-color: rgb(99, 112, 243);
font: 13pt &quot;Comfortaa&quot;;
color: rgb(56, 64, 139);</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_c2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 20px;
border-width : 2px;
border-style:solid;
border-radius: 20px;
  min-height: 20px;
  min-width: 20px;
background-color: rgb(233, 240, 250);
border-color: rgb(99, 112, 243);
font: 13pt &quot;Comfortaa&quot;;
color: rgb(56, 64, 139);</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_c0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 20px;
border-width : 2px;
border-style:solid;
border-radius: 20px;
  min-height: 20px;
  min-width: 20px;
background-color: rgb(233, 240, 250);
border-color: rgb(99, 112, 243);
font: 13pt &quot;Comfortaa&quot;;
color: rgb(56, 64, 139);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
